examples/gnrc_border_router/README.md

```diff
@@ -60,3 +60,99 @@ for further help.
 [1] https://tools.ietf.org/html/rfc1055
 
 [2] https://github.com/contiki-os/contiki/blob/master/tools/tunslip.c
+
+
+# gnrc_border_router with ethos
+A new setup is possible on which only one serial interface is needed. This is done through the ethos (Ethernet Over Serial) driver that multiplexes serial data between ethernet packets and shell commands.
+
+## Requirements
+This functionality works only in Linux machines, Mac support will be added in the future (lack of `tap` interface), and Windows support should possible by setting up the needed `tap` interface with an external program (not tested).
+
+## Setup
+Take care about the modification of the `Makefile`, since by default SLIP is enabled.
+In order to select ethos as the driver, you should modify some lines as following:
+```make
+#CFLAGS += -DSLIP_UART=$(SLIP_UART)
+#CFLAGS += -DSLIP_BAUDRATE=$(SLIP_BAUDRATE)
+CFLAGS += '-DETHOS_UART=UART_DEV(0)' -DETHOS_BAUDRATE=115200 -DUSE_ETHOS_FOR_STDIO
+```
+Then deactivate the slip module and activate ethos
+```make
+# Include SLIP package for IP over Serial communication
+#USEMODULE += gnrc_slip
+USEMODULE += ethos gnrc_netdev2
+```
+This will allow to compile the BR firmware using ethos.
+Thus:
+```bash
+$ make clean all flash
```

Comment:
`s/^$ //` if you syntax highlight as bash (otherwise `$` will be parsed as variable I think)